### ISSUE TITLE
Warning: `Overfull \hbox (30.0pt too wide) in paragraph at lines 11--11 []`

### ISSUE THREAD

**Issue by jasperhabicht** [treyhunner/resume#11]
I noticed that there is a warning regard to document command when use it like `\begin{document}`. The message is `Overfull \hbox (30.0pt too wide) in paragraph at lines 11--11 []`. Do you have any idea how to fix it?

Version: Latexmk, John Collins, 17 Mar. 2022. Version 4.77, version: 4.77

_Originally posted by @passos in https://github.com/treyhunner/resume/issues/9#issuecomment-1325842857_

**Comment by jasperhabicht** [comment]
A solution for this is to replace the original definition of `\printname` by the following code (and thus matching the code to that of `\printaddress` and also considering [my solution to issue #8](https://github.com/treyhunner/resume/issues/8#issuecomment-1321217715)):

```diff
   \def \printname {
     \begingroup
-      \hfil{\MakeUppercase{\namesize\bf \@name}}\hfil
+      \centerline{\textbf{\MakeUppercase{\namesize\@name}}}
-      \nameskip\break
     \endgroup
+    \par
+    \nameskip
   }
```

**Comment by treyhunner** [comment]
Fixed in #10
